test/ServerIntegrationTest.php

```diff
@@ -43,5 +42,7 @@ public function testPassesBufferLevelToSapiStreamEmitter()
         );
         $server->setEmitter($emitter->reveal());
         $server->listen();
+
+        ob_end_clean();
```

Comment:
Discovered why when removing locally; PHPUnit flags it as a risky test due to the fact it did not close its own output buffers.